testing/oak_echo_raw_bin/README.md

```diff
@@ -0,0 +1,4 @@
+# Echo Baremetal
```

Comment:
What is the difference between the raw and non-raw binaries? Also we should not use baremetal, freestanding is the correct term.

What is the binary expected to do?

Also, perhaps the other binary should be renamed to clarify that it actually uses the IDL, while this one doesn't (is that correct)?

Reply:
Submitted a PR, added comments in: https://github.com/project-oak/oak/pull/3410